src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -51,34 +66,67 @@ public void ConfigureServices(IServiceCollection services)
                 options.UseSqlite(
                     Configuration.GetConnectionString("DefaultConnection")));
     #endif
-            
             services.AddIdentity<IdentityUser, IdentityRole>(options => options.Stores.MaxLengthForKeys = 128)
                 .AddEntityFrameworkStores<ApplicationDbContext>()
                 .AddDefaultUI()
                 .AddDefaultTokenProviders();
-
-#elif (OrganizationalAuth || IndividualB2CAuth)
-            services.AddAuthentication(sharedOptions =>
+#elif (OrganizationalAuth)
+            services.AddAuthentication(AzureADDefaults.AuthenticationScheme)
+                .AddAzureAD(options => Configuration.Bind("AzureAd", options));
+#if (MultiOrgAuth)
+            
+            services.Configure<OpenIdConnectOptions>(AzureADDefaults.OpenIdScheme, options =>
             {
-                sharedOptions.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-                sharedOptions.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
-            })
-    #if (OrganizationalAuth)
-                .AddAzureAd(options => Configuration.Bind("AzureAd", options))
-    #elif (IndividualB2CAuth)
-                .AddAzureAdB2C(options => Configuration.Bind("AzureAdB2C", options))
-    #endif
-            .AddCookie();
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    // Instead of using the default validation (validating against a single issuer value, as we do in
+                    // line of business apps), we inject our own multitenant validation logic
+                    ValidateIssuer = false,
 
-#endif
-            services.Configure<CookiePolicyOptions>(options =>
-            {
-                // This lambda determines whether user consent for non-essential cookies is needed for a given request.
-                options.CheckConsentNeeded = context => true;
-                options.MinimumSameSitePolicy = SameSiteMode.None;
+                    // If the app is meant to be accessed by entire organizations, add your issuer validation logic here.
+                    //IssuerValidator = (issuer, securityToken, validationParameters) => {
+                    //    if (myIssuerValidationLogic(issuer)) return issuer;
```

Comment:
See earlier comment.

Reply:
Discussed offline, we don't really care as users need to customize this after creating the template.